config/values/bursaries.yml

```diff
@@ -19,3 +19,4 @@ bursaries:
     physics: £28,000
     religiouseducation: £10,000
     nonuk: £28,000
+  fundingwindow: between September 2024 and July 2025
```

Comment:
Looks good!

My only (very minor) comment would be whether to simplify the variable name eg `bursaries_fundingdates` or `bursaries_dates` but up to you